examples/pubsub/tutorial_pubsub_mqtt_publish.c

```diff
@@ -272,7 +272,7 @@ addWriterGroup(UA_Server *server, char *topic, int interval) {
     transportSettings.content.decoded.type = &UA_TYPES[UA_TYPES_BROKERWRITERGROUPTRANSPORTDATATYPE];
     transportSettings.content.decoded.data = &brokerTransportSettings;
 
-    writerGroupConfig.transportSettings = transportSettings;
+    UA_ExtensionObject_copy(&transportSettings, &writerGroupConfig.transportSettings);
```

Comment:
Why is this coppy needed? The config structures should be deep-copied e.g. witihin "UA_Server_addWriterGroup"

Reply:
It is not needed. Removed it